client/src/app/login/login.component.css

```
@@ -0,0 +1,85 @@
+* {
+    font-family: 'JetBrains Mono', monospace;
+}
+
+.login-form-container {
+        flex-direction: column;
+        justify-content: center;
+        text-align: center;
+        width: 434px;
+        margin: 15% auto;
```

I think it is better to leave margin: auto; So it doesn't change height position when resizing the screen.
```suggestion
        margin: auto;
```